solidity/deploy/07_transfer_ownership.ts

```diff
@@ -0,0 +1,17 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, helpers } = hre
+  const { deployer, governance } = await getNamedAccounts()
```

Comment:
This will be a smart contract similar to `WalletRegistryGovernance`. We don't have it yet though.